docs/index.html

```diff
@@ -1026,8 +1026,177 @@ <h3 id="col-bleed-y">.col-bleed-y <a href="#col-bleed-y">&#35;</a></h3>
 
             <section class="container">
                 <h2 id="visibility-helpers">Visibility helpers <a href="#visibility-helpers">&#35;</a></h2>
-                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. resize the viewport to see them in action. The breakpoints can be changed if you're building the css yourself by building the SCSS files.</p>
-
+                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. 
+                    <br>The <code>.hidden-*-up</code> classes hide the element when the viewport is at the given breakpoint or wider, 
```

Comment:
Would prefer to close the paragraph here and present these lines as a `<ul>` with each example as a `<li>`

Reply:
It's a good idea, it will be clearer.